src/cloud-api-adaptor/test/e2e/README.md

```diff
@@ -48,6 +48,38 @@ To leave the cluster untouched by the execution finish you should export `TEST_T
 
 To use existing cluster which have already installed Cloud API Adaptor, you should export `TEST_INSTALL_CAA=no`.
 
+## Attestation and KBS specific
+
+We need artificats from trustee when do attestation test, to prepare trustee, do as following. 
+
+```
+pushd ${cloud-api-adaptor}/test/e2e
+git clone https://github.com/confidential-containers/trustee.git
+pushd trustee
+git checkout $(../../../hack/yq-shim.sh '.git.kbs.reference' ../../../versions.yaml)
+pushd kbs
+make CLI_FEATURES=sample_only cli
+popd
+popd
+popd
+```
+
+We need build and use the PodVM image with `AA_KBC=cc_kbc` enabled, for example:
+```
+pushd ${cloud-api-adaptor}
+AA_KBC=cc_kbc make podvm-builder podvm-binaries podvm-image
+popd
+```
+Then extract the PodVM image and use it following [extracting-the-qcow2-image](../../podvm/README.md#extracting-the-qcow2-image)
+
+To deploy the KBS service and test attestation related cases, export following variables like:
+```
+export DEPLOY_KBS=yes
+export KBS_IMAGE="ghcr.io/confidential-containers/staged-images/kbs"
+export KBS_IMAGE_TAG="dc01f454264fb4350e5f69eba05683a9a1882c41"
+````
+Where the value of `KBS_IMAGE` and `KBS_IMAGE_TAG` should be same as described in [versions.yaml](../../versions.yaml)
```

Comment:
I'm not against this as is, but wonder if it's worth using something like `export KBS_IMAGE_TAG=$(./hack/yq-shim.sh '.git.kbs.reference' ./versions.yaml)` here as well rather than that hard-coded version and pointing them to the yaml? I am aware there is a complication in making sure the use is in the correct directory when running this though.

Reply:
Fixed.